include/RAJA/config.hxx.in

```diff
@@ -154,102 +154,106 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //
 //     RAJA_INLINE - macro to enforce method inlining
 //
-//     RAJA_ALIGN_DATA(<variable>) - macro to express alignment of data, 
-//                              loop bounds, etc.
+//     RAJA_SIMD - macro to apply compiler-specific pragma that encourages
+//                 compiler to SIMD vectorize loop even if it may not
+//                 normally
+//
+//     RAJA_SIMDFORCE - macro to apply compiler-specific pragma that forces
```

Comment:
My $0.02:

* `seq_exec` should be strictly sequential -- it's clear what it does and the programmer cannot shoot themselves in the foot
* `omp_parallel_for_exec` and `cuda_exec` force parallelization, which differs from the Parallelism TS.
* I like the idea of `simd_force_exec` because the naming is clear, but it's also an outlier in terms of naming.
* In the long term, and since I've been using RAJA for tuning codes, having options like `hint` or `force` for SIMD would be the most expressive: `RAJA::simd_hint_exec>` or `RAJA::simd_force_exec`

For compatibility, `RAJA::simd_exec` aliases to `RAJA::simd_hint_exec` to preserve default safe code generation.